### ISSUE TITLE
Include interactive demo in README file

### ISSUE THREAD

**Issue by martinzugnoni** [inveniosoftware/dictdiffer#127]
We've been using `dictdiffer` lately and I think it would be great to provide some sort of interactive online demo for the library. I helped adding it to `jsonschema` project, you can see it running here:
https://github.com/Julian/jsonschema#demo

Do you think this will help users learn the library easier? If you want, I can create a PR to include a demo link in `dictdiffer` as well.

**Comment by jirikuncar** [comment]
@martinzugnoni it would be definitely nice to **include more examples in README**. If you have time you can help us by submitting a PR.

Personally, I am not sure if linking to a 3rd party service is a good idea, but if we go this path we should consider linking to similar services like mybinder.org too.

**Comment by martinzugnoni** [comment]
Absolutely! The more options we give to the user the more chances we have that they access to such interactive demos, and thereby the easier/sooner they will have a feeling of how the library works.

I will submit a PR in the upcoming days/weeks and wait for feedback. Thanks for your reply.
